test/foundry/FleekERC721/FleekERC721.ap.t.sol

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: MIT
```

Comment:
can we rename this file to `FleekERC721.accessPoint.t.sol`? It's more descriptive.

Reply:
Same for `FleekERC721.access.t.sol`.
Maybe we can come up with a better name.